src/main/java/com/ua/accommodation/service/impl/BookingServiceImpl.java

```diff
@@ -121,4 +130,18 @@ private void checkAccommodationAvailability(
                             + checkInDate + " to the " + checkOutDate);
         }
     }
+
+    private void publishEvent(Booking booking) {
+        StringBuilder builder = new StringBuilder();
+        String message = builder.append("Booking update!\n")
+                .append("Id: " + booking.getId() + "\n")
```

Comment:
The only thing that we using append and string concatenation altogether here.
Maybe it`s better to use concatenation since we don't have a loop here.